Source/Scene/OctahedralProjectedCubeMap.js

```diff
@@ -302,11 +303,28 @@ OctahedralProjectedCubeMap.prototype.update = function (frameState) {
     return;
   }
 
+  var defines = [];
+  // Datatype is defined if it is a normalized type (i.e. ..._UNORM, ..._SFLOAT)
```

Comment:
Is this the right fix?

```suggestion
  // Datatype is defined if it is a normalized type (i.e. ..._UNORM)
```

Reply:
Maybe we could discuss this and the comment below offline because I'm a little confused about when `pixelDatatype` is defined and how the behavior differs for unorm vs. float textures.

Reply:
Both UNORM and SFLOAT outputs from the Khronos IBL sampler output rgb values in [0, 1], so I push the normalized "define" in both cases. In the shader, the behavior should be the same for UNORM and SFLOAT but it might be possible to get a cubemap with unnormalized rgb (i.e. Filament sampler).

Reply:
> Both UNORM and SFLOAT outputs from the Khronos IBL sampler output rgb values in [0, 1]

Is this true? I would've thought HDR float textures would go beyond the [0, 1] range. The shader is fine though, the "normalized" naming just trips me up.